client/src/components/Page.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import styled from 'styled-components';
+import Details from './Details.jsx';
+import Dropdown1 from './Dropdown1.jsx';
+import Stars from './Stars.jsx';
+import Material from './Material.jsx';
+import NameOfItem from './NameOfItem.jsx';
+import $ from 'jquery';
+
+const Wrapper = styled.div`
+  display: inline-block;
+`;
+const Title = styled.h1`
+  font-family: 'Graphik Webfont', -apple-system, BlinkMacSystemFont, 'Roboto',
+    'Droid Sans', 'Segoe UI', 'Helvetica', Arial, sans-serif;
+  font-weight: 300;
+  font-size: 15px;
+  line-height: 28px;
+  color: black;
+`;
+
+const SalesTitle = styled.h2`
+  font-family: -apple-system, BlinkMacSystemFont, 'Roboto', 'Droid Sans',
+    'Segoe UI', 'Helvetica', Arial, sans-serif;
+  font-size: 14px;
+  line-height: 1.4;
+`;
+export default class Page extends React.Component {
+  constructor(props) {
+    super(props);
+    (this.state = {
+      product: [],
+      apiData: [],
+    }),
+      (this.getDataFromDB = this.getDataFromDB.bind(this));
+    this.getDataFromApi = this.getDataFromApi.bind(this);
+  }
+
+  componentDidMount() {
+    this.getDataFromDB(this.props.id);
+    this.getDataFromApi(this.props.id);
+  }
+
+  getDataFromDB(productId) {
+    const { product } = this.state;
+    $.ajax({
+      url: `/itemDetails/${productId}`,
+      method: 'GET',
+      contentType: 'application/json',
+      success: (productData) => {
+        console.log('res data ----->', productData);
+        this.setState({
+          product: productData,
+        });
+      },
+      error: (xhr, status, error) => {
+        console.log('err', xhr, status, error);
+      },
+    });
+  }
+
+  getDataFromApi() {
+    const pathname = window.location.pathname;
+    const productId = pathname.substring(pathname.lastIndexOf('/') + 1) || 3;
```

Comment:
I deleted these lines a long time ago, but since I deleted the entire folder with components and restored them from the depths of my laptop, some files still contain old, uncorrected code, I probably missed it.
It doesn't matter anymore. Thank you for paying attention to this!